_topics/en/op_checktemplateverify.md

```diff
@@ -139,15 +139,16 @@ assures several receivers that they can each be paid.  This two-step
 process can probably be used anywhere payment batching is an option
 but it can likely reduce fees even further than payment batching.
 
-Later versions of the proposal placed greater emphasis on other
+The original proposal also emphasized other
```

Comment:
> At the time I felt I really needed people to grok congestion control as a start, in order to start thinking about how the other applications could work

This is exactly what I observed and what I wrote.  The original proposal (via your email) and discussion emphasized congestion control.  Later publications by you emphasized other applications.

> I think the current "greater emphasis" text serves to build on a narrative that CTV was pitched as "only" for congestion control, and it was very limited, and other uses were an afterthought that only came much later to show it could do more than 1 thing, where the truth is that it was presented always with multiple applications.

I'm sorry that there's a narrative that contains provably false statements and that you think our text, despite being accurate AFAICT, contributes to that.

Reply:
FWIW i don't think it's intentional at all and apoliogize thoroughly if you've read it as such.

I meant my top post quite literally -- to the extent people are trying to get an accurate history of the development of covenants, there are some persistant myths worth (to me, maybe others) cleaning up.